### PR TITLE
SG-30223 Reformat code examples to prevent text overflow

### PR DESCRIPTION
**Pull request by dsauve-adsk**
Preview of the changes ( the same changes have been applied to the two functions )
Screenshot 2023-03-13 at 9 44 23 AM

### CODE REVIEW

File: shotgun_api3/shotgun.py [hunk 1]
```diff
@@ -850,7 +850,10 @@ def find_one(self, entity_type, filters, fields=None, order=None, filter_operato
             Defaults to ``["id"]``.
         :param int order: Optional list of fields to order the results by. List has the format::
 
-            [{'field_name':'foo', 'direction':'asc'}, {'field_name':'bar', 'direction':'desc'}]
+                [
+                 {'field_name':'foo', 'direction':'asc'},
+                 {'field_name':'bar', 'direction':'desc'}
+                ]
```
Comment (carlos-villavicencio-adsk) [inline shotgun_api3/shotgun.py:856]: Maybe the indentation can be improved:
```suggestion
                [
                    {'field_name':'foo', 'direction':'asc'},
                    {'field_name':'bar', 'direction':'desc'},
                ]
```

**Review by carlos-villavicencio-adsk — COMMENTED** [review]
I'd suggest adding indentation to multiline lists and dicts as `black` would format them.

File: shotgun_api3/shotgun.py (hunk 1 above)
Comment (dsauve-adsk) [reply shotgun_api3/shotgun.py]: Just push that suggestion...
Screenshot 2023-03-13 at 4 46 13 PM

**Review by carlos-villavicencio-adsk — APPROVED** [review]
LGTM

**Pull request merged**